db/db_test_util.h

```diff
@@ -49,10 +49,7 @@
 #include "util/mutexlock.h"
 
 #include "util/string_util.h"
-// SyncPoint is not supported in Released Windows Mode.
```

Comment:
What do you mean? The entire file is behind a guard:

https://github.com/facebook/rocksdb/blob/master/util/sync_point.h#L28
https://github.com/facebook/rocksdb/blob/master/util/sync_point.cc#L17